app/views/content/help-and-advice.md

```diff
@@ -11,22 +11,27 @@ calls_to_action:
     name: chat
     arguments: {}
 ---
+## Get an adviser
 
-## Get a teacher training adviser
+Advisers provide free, one-to-one support by phone, text or email. They're all experienced, former teachers and able to answer any questions you have.
 
-Teacher training advisers provide free, one-to-one support by phone, text or email wherever you are in your journey into teaching. They are experienced, former teachers and able to answer any questions you have. You will have your own dedicated adviser to support you at every step as you make important decisions about your future.
+You will have your own dedicated adviser to support you.
 
-They can help you to:
+### Teacher training advisers
+
+A <a href="/teacher-training-advisers">teacher training adviser</a> can help you if you're a final year undergraduate student or if you already have a degree. They can help you:
 
-- get classroom experience
-- find out what teaching is really like
 - understand more about teacher training courses
-- understand the funding and support available to you
-- find the right courses to apply for
+- find funding and support available to you
 - make a successful application
-- prepare for your interview
 
-<a href="https://adviser-getintoteaching.education.gov.uk/" class="button">Get your dedicated adviser</a>
+### Explore teaching advisers
+
+An <a href="/explore-teaching-advisers">explore teaching adviser</a> can help you if you're an undergraduate student not in your final year of study. They can help you:
```

Comment:
```suggestion
An <a href="/explore-teaching-advisers">explore teaching adviser</a> can help you if you're an undergraduate student and not in your final year of study. They can help you:
```

Happy for you to disagree but I wonder if adding in 'and' helps with the readability of this sentence